### DISCUSSION TITLE
There is a major difference in saturation between using firmware GPLog and setting Log through Labs (Hero 12)

### DISCUSSION THREAD

**Discussion by Squinkz** [gopro/labs#1107] [General]
I have been using Log set through Labs since I first got my Hero 12 (logb=400, wide=1), but I figured I should probably check out the stock implementation. I ran some tests and to my surprise, the stock GPLog is considerably less saturated than the Labs implementation.

Here are some reading from the vectorscope in Resolve (I am working non color managed, so this should be a direct reading of the file, no input DRTS or other transforms): https://i.imgur.com/MRu1WME.png

Is this a known issue? For what it's worth I generally use the DCTL provided by Xtremestuff, and that provides almost 1:1 results vs shooting in Protune Flat when shooting in Log set through GoPro Labs. The stock GPLog needs saturation boosted a varying amount (depends on the scene/exposure) to match Protune Flat. (I find GoPro's provided LUT provides unsatisfactory results in both cases.)

**Comment by dnewman-gpsw** [comment]
I am not finding that. When using WIDE=1 and LOGB=400, you need to also combine that with cF (Color Flat), sharpness Low and ISO Max 400. The default GPLog includes those settings.  The key one is color Flat, as color Natural and Vibrant include additional saturation and tonemapping.

**Reply by Squinkz** [reply]
https://imgur.com/a/jDVHSBy

These are the QR codes I was using to do the tests, I'm fairly certain everything was on flat. ISO was limited to 200 and shutter was on auto (lighting was constant so I didn't set a specific shutter speed for the tests).

Perhaps I'll do a complete Labs reset and try again tomorrow.

**Reply by dnewman-gpsw** [reply]
This might be an order of operations problem with those QR codes.  You enable GPLog that use settings GPLog doesn't have (cF) and then disable LOGB and WIDE, which could mess with the GPLog setup.  That will not be guaranteed to work.  All I did, turn on the camera, Shoot GP-Log, switched to a standard preset, then scanned cF*WIDE=1*LOGB=1, and got the same results.

**Reply by dnewman-gpsw** [reply]
I just retested. You are correct.  I don't know how I got my first results, but I'm now seeing a saturation difference.

**Reply by RikshaDriver** [reply]
This sounds oddly familiar:

https://github.com/gopro/labs/discussions/636

That discussion was pre firmware 2.0 HERO12

I no longer have the 12, but recent testing with H13 and H11 side by side shows GP-Log on H13 and LOGB400 on 11 are similar.

Also LOGB400 on 13 is oversaturated compared to dedicated GP-Log profile.

**Reply by Squinkz** [reply]
Alright so I cleared all my settings out and reran the tests, then confirmed all the correct settings were applied in the video metadata.

Got similar results, here are my test shots (samples in raw log, rec 709 conversion with DCTL and official LUT, and protune flat).

https://imgsli.com/MzAxODgz
https://imgsli.com/MzAxODg2
https://imgsli.com/MzAxODg3
https://imgsli.com/MzAxODg4
